### ISSUE TITLE
The tool sniff can display Qmenu (popup menu) widget, but if the qmenu is hided the accessible name will disappear.

### ISSUE THREAD

**Issue by wangyingtaodeepin** [vhumpa/dogtail#1]

**Comment by vhumpa** [comment]
Indeed, that means likely that the UI object has been deleted from the a11y tree, so you need to keep it open in order to work with the menu. For the purposes of development (when working with sniff on the same desktop ends up in menu closure), I suggest using dogtail directly best via ipython, best ssh'ed into a VM where the app you're interested in is running. This way you will have no interactions between the dev (host) and test (guest) instance. Running 'dogtail-run-headless-next bash' when ssh'ed in the guest system will spawn a bash line inside the target environment you can use. Also sorry for the late answer, we're mainly operating on https://gitlab.com/dogtail/dogtail/ , this being a fork repo.
